components/Layout.tsx

```diff
@@ -1,14 +1,16 @@
 import { FC, PropsWithChildren } from 'react';
 import Sidebar from './SideBar';
 import NavBar from './NavBar';
+import BottomNavBar from './BottomNavBar';
 
 const Layout: FC<PropsWithChildren> = ({ children }) => {
   return (
     <>
       <NavBar />
       <Sidebar />
-      <main className="bg-white dark:bg-ocf-black flex flex-col items-center justify-start px-10">
+      <main className="bg-white dark:bg-ocf-black flex flex-col items-center justify-start px-10 mb-[100px]">
```

Comment:
lets remove this margin bottom—it seems to not be doing anything, and removing it only makes the pages work better on my end

```suggestion
      <main className="bg-white dark:bg-ocf-black flex flex-col items-center justify-start px-10">
```